php/libraries/Utility.class.inc

```diff
@@ -747,41 +764,14 @@ class Utility
      * @return array of study visits in the format array('VL' => 'VL')
      *         where VL is the visit label
      *
-     * @note       THIS FUNCTION USES STRING CONCATENATION TO GENERATE AN
-     *             SQL STATEMENT.
-     *             IT IS VULNERABLE TO AN SQL INJECTION ATTACK.
```

Comment:
I removed this comment because I do not think it is vulnerable:

* Only fixed strings are use to extend the query. 
* pselect is used which should still prepare the only part of the query that can be supplied by a user
* The `int` type hint will throw an error on anything that isn't numeric or null and I don't think it's possible to create any kind of SQL injection attack with integers only.

Reply:
The comment was likely written before it was switched to pselect..

(Even if it wasn't, changing the function to just throw an exception unconditionally also makes it non-vulnerable..)

Reply:
Fair enough, I figured it was probably a historical thing. I figured somebody would ask about it so I wanted to lay out why this change is safe.